main.py

```diff
@@ -0,0 +1,43 @@
+from fastapi import FastAPI, Request
```

Comment:
What is `fastapi`? A developer wanting to run your code should know what third-party packages he has to install. Add `requirements.txt` file with the requirements for the project. Read on the web about how this file should look.